plugin/inc/installsteps/handleprofilerightsinstallstep.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace GlpiPlugin\Iservice\InstallSteps;
+
+class HandleProfileRightsInstallStep
+{
+    const CLEANUP_ON_UNINSTALL = true;
+
+    public static function do(): bool
+    {
+        global $DB;
+
+        $table = 'glpi_profilerights';
+        $superAdminProfileId = \Profile::getSuperAdminProfilesId();
+        $rightsValues = include __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'rights.config.php';
```

Comment:
I read somewhere that the use of DIRECTORY_SEPARATOR is absolutely not necessary. Windows can handle `/` in the path very well.
```suggestion
        $rightsValues = include PLUGIN_ISERVICE_DIR . '/inc/rights.config.php';
```